src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -980,10 +1001,11 @@
 				<VisualState x:Name="MultiselectDisabled" />
 				<VisualState x:Name="MultiselectEnabled">
 					<VisualState.StateTriggers>
-						<triggers:IsEqualStateTrigger Value="{x:Bind AppModel.MultiselectEnabled, Mode=OneWay}" To="True" />
+						<triggers:IsEqualStateTrigger Value="{x:Bind Preferences.ShowSelectionCheckboxes, Mode=OneWay}" To="True" />
```

Comment:
We probably shouldn't bind directly to the service, it would be better to create a property in the ViewModel or code behind in this case.

Reply:
Fair point, AppModel is now handling that for us.

Reply:
AppModel works for now, at some point we should have a ViewModel and Model for the layout.